<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Welcome</class>
 <widget class="QWidget" name="Welcome">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Trading Simulator</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>360</width>
     <height>360</height>
    </rect>
   </property>
   <widget class="QWidget" name="Home">
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>150</y>
       <width>269</width>
       <height>120</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>150</number>
      </property>
      <item>
       <widget class="QPushButton" name="About">
        <property name="text">
         <string>About</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Continue">
        <property name="text">
         <string>Continue</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>221</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="resource.qrc">:/TradingSimulator/evolutionCours/icons/welcome.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="MainMenu">
    <widget class="QToolButton" name="Candlestick">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>140</width>
       <height>140</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="toolTip">
      <string>Display</string>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QToolButton" name="Simulation">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>140</width>
       <height>140</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="toolTip">
      <string>Simulation</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>180</y>
       <width>140</width>
       <height>140</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QToolButton" name="Quit">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>180</y>
       <width>140</width>
       <height>140</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="toolTip">
      <string>Quit</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Mode">
    <widget class="QToolButton" name="Manuel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>140</width>
       <height>140</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="toolTip">
      <string>Manuel mode</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QToolButton" name="Step">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>140</width>
       <height>140</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="toolTip">
      <string>Step by step</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QToolButton" name="Auto">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>180</y>
       <width>140</width>
       <height>140</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="toolTip">
      <string>Auto Mode</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QToolButton" name="Back">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>180</y>
       <width>140</width>
       <height>140</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="toolTip">
      <string>Back</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Candle">
    <widget class="QDialogButtonBox" name="dispaly_dialog">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>300</y>
       <width>164</width>
       <height>32</height>
      </rect>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>291</width>
       <height>191</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Currency 1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="Currency1"/>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Currency 2</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="Currency2"/>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>File Path</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="linePath"/>
        </item>
        <item>
         <widget class="QToolButton" name="filePath">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
